features/mbedtls/importer/Makefile

```diff
@@ -62,6 +71,23 @@ rsync:
 	cp $(MBED_TLS_DIR)/LICENSE $(TARGET_PREFIX)
 	cp $(MBED_TLS_DIR)/apache-2.0.txt $(TARGET_PREFIX)
 	#
+	# Create Mbed Crypto target folder
+	mkdir -p $(TARGET_PREFIX_CRYPTO)
+	#
+	# Copying Mbed Crypto into Mbed OS..
```

Comment:
I think what's going on here is a bit opaque to outsiders and other teams. There needs to be explanation here, (or pointers to documentation elsewhere) what these directories are and what they mean.

We need to explain what SPE, NSPE are, and what the other directories are and what they mean.

And most of all, we need to explain *why* we have so many directories. It will seem bizarre and unnecessarily complicated to outsiders.

Reply:
Since this directory hierarchy is exist in mbed-os I don't know how adding explanation here will help outsiders.
#8680 PR description explains that but not the merged code.
This PR: https://github.com/ARMmbed/mbed-os-5-docs/pull/815 explains more about that and i think it will be helpful for outsiders

Reply:
Ok - Mbed TLS is my project that I work on daily, and these directories mean nothing to me. What do they do? What are they for? There's no explanation.

I think we need to either:
 * add comments to explain these directories, either in this file or in another file in this directory
 * tell me where there's other existing documentation that explains the structure elsewhere in the repository
 * add a comment that points the reader to documentation somewhere else

Reply:
I added the required documentation in the importer.